public_html/src/userObject.php

```diff
@@ -56,13 +56,20 @@ public function __construct(array $vars, $fromDB){
 			$this->toolAdmin = 0;
 			$this->checkuser = 0;
 			$this->developer = 0;
-			$this->useSecure = isset($vars['useSecure']);
+			$this->useSecure = !isset($vars['useSecure']);

```

Comment:
We should really just nix `useSecure`; sites with sensitive data should not allow usage via HTTP in 2015 (almost 2016).

Can we just redirect HTTP to HTTPS and I can rip this out entirely? :)